game.py

```diff
@@ -58,6 +63,18 @@ def get_calculation_class():
         else:
             return Subtraction(1, 20)
 
+    @staticmethod
+    def save_stats(username: str, correct_count: int, incorrect_count: int):
+        today = date.today()
+        content = [today, username, correct_count, incorrect_count]
+        file_exists = os.path.isfile('statistics.csv')
+        with open('statistics.csv', 'a', newline='\n') as csvfile:
+            headers = ["Day", "Username", "Correct count", "Incorrect count"]
+            writer = csv.writer(csvfile)
+            if not file_exists:
+                writer.writerow(headers)
+            writer.writerow(content)
+
```

Comment:
This code looks very good, but we still need a unit test for it so that we can be sure that it works correctly.